docs/getting-started/quickstart.md

```diff
@@ -46,6 +46,21 @@ directus/directus
 
 _Learn more about self-hosting Directus in our [Self-Hosted Quickstart](/self-hosted/quickstart)._
 
+== NPM Installation
+
+For quick prototyping and local developments, you can initialize a new Directus project using the CLI command:
+`npm init directus-project`.
+
+This will prompt you to choose a database client and path. Go ahead to also create your first admin user.
+
+To start up Directus locally, run `npx directus start`.
+
+::: danger Docker is Recommended for Production
+
+While this method is suitable for quick tests and prototypes, it is not recommended for production. We recommend you use
+[Docker](/self-hosted/quickstart) because it removes environment-specific quirks that can lead to Directus not running
+properly.
+
 :::
```

Comment:
I'm not sold on this narrative. It's not a non-prod/prod split, it's a potential complexity split. 

```suggestion
You can initialize a new Directus project using the CLI command:
`npm init directus-project`.

Follow prompts to configure your database and create your first admin user.

To start Directus locally, run `npx directus start`.

:::danger Docker is Recommended

While this method works and is maintained, it is not recommended. Docker removes environment-specific quirks that can lead to Directus not running
properly in some contexts. 

:::
```

Removing link to Self-Hosted Quickstart too as it's the previous tab in the box this text displays in. We'll probably want @rijkvanzanten to chime in with desired wording too.

Reply:
The current state LGTM!